### ISSUE TITLE
Date Chooser hides behind loading screen

### ISSUE THREAD

**Issue by derNarr** [MegaMek/mekhq#736]
### Environment
What version of MekHQ does your issue apply to? At least `mekhq-0.43.10-RC4`
What operating system are you using?  `Linux`
What java version are you using?
```
openjdk version "1.8.0_171"
OpenJDK Runtime Environment (build 1.8.0_171-8u171-b11-1~deb9u1-b11)
OpenJDK 64-Bit Server VM (build 25.171-b11, mixed mode)
```
### Description
The date popup (Date Chooser) hides behind the unmovable loading screen, after tapping in and out. Therefore, if you tap in and out it is not possible to choose a date anymore and the program has to be forcefullly closed.

### How to reproduce
1. Start MekHQ
2. Click 'Start a New Campaign'
3. Wait until the Date Chooser pups up
4. Tab in and out
5. 'Date Chooser' is now behind the loading screen.

### Workaround
Simply choose a date without doing anything else in between.

### Additional Observation
The 'Choose Game Preset' popup / window does not have this problem.

**Comment by neoancient** [comment]
I could not reproduce this.

**Comment by firefly2442** [comment]
I also was not able to reproduce on Linux with 61dc6d8fbcce82c4b3f00e7d29341761f0f07fa1.  However, I'm not using OpenJDK.

```
java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)
```

**Comment by sixlettervariables** [comment]
Fixed in 0.45.3.
